src/sql/workbench/contrib/tsgops/browser/tsgopsActions.ts

```diff
@@ -0,0 +1,64 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { localize } from 'vs/nls';
+import { Action } from 'vs/base/common/actions';
+
+import { IWorkbenchLayoutService } from 'vs/workbench/services/layout/browser/layoutService';
+import { IActivityBarService } from 'vs/workbench/services/activityBar/browser/activityBarService';
+
+export class HidePanel extends Action {
+	static readonly ID = 'workbench.action.hidePanel';
+	static readonly LABEL = localize('hidePanel', "Hide the panel...");
+
+	constructor(
+		id: string = HidePanel.ID,
+		label: string = HidePanel.LABEL,
+		@IWorkbenchLayoutService private readonly layoutService: IWorkbenchLayoutService
+	) {
+		super(id, label);
+	}
+
+	override async run(): Promise<void> {
+		this.layoutService.setPanelHidden(true);
+	}
+}
+
+export class HideSettings extends Action {
+	static readonly ID = 'workbench.action.hideSettings';
+	static readonly LABEL = localize('hideSettings', "Hide the settings icon...");
```

Comment:
nit: Windows UI guidelines says that "..." at end of command implies that the command will show a follow-up dialog/UX.  That doesn't seem to be the case here so I'm not sure why the command labels have a trailing "...".